src/components/experiment-tracking/run-dataset/run-dataset.js

```diff
@@ -226,8 +239,9 @@ function buildDatasetDataMarkup(
   const isPlotlyDataset = getShortType(datasetType) === 'plotly';
   const isImageDataset = getShortType(datasetType) === 'image';
   const isJSONTrackingDataset = getShortType(datasetType) === 'JSONTracking';
-  const isMetricsTrackingDataset = getShortType(datasetType) === 'metricsTracking';
-  const isTrackingDataset = isJSONTrackingDataset || isMetricsTrackingDataset
+  const isMetricsTrackingDataset =
+    getShortType(datasetType) === 'metricsTracking';
+  const isTrackingDataset = isJSONTrackingDataset || isMetricsTrackingDataset;
```

Comment:
What are these changes?

Reply:
i think just pretify. we did this change in an old PR - the json view release.